docs/.vuepress/theme/components/AposFeedback.vue

```diff
@@ -0,0 +1,162 @@
+<template>
+  <!-- Based on https://github.com/ipfs/ipfs-docs-v2/blob/2a019d7c8abe99c21a3150752915ed278a4592ff/docs/.vuepress/theme/components/Feedback.vue -->
```

Comment:
This is mostly all courtesy of the `ipfs-docs-v2`, though I changed presentation and removed a few bits we don't need. If any seems overly superfluous, we can remove it, but we'll also likely be doing cosmetic work later on anyway.